client/blocks/reader-share/social.jsx

```diff
@@ -14,17 +14,17 @@ import { recordReaderTracksEvent } from 'calypso/state/reader/analytics/actions'
  * Local variables
  */
 const actionMap = {
-	twitter( post ) {
+	x( post ) {
 		const baseUrl = new URL( 'https://twitter.com/intent/tweet' );
 		const params = new URLSearchParams( {
 			text: post.title,
 			url: post.URL,
 		} );
 		baseUrl.search = params.toString();
 
-		const twitterUrl = baseUrl.href;
+		const xUrl = baseUrl.href;
 
-		window.open( twitterUrl, 'twitter', 'width=550,height=420,resizeable,scrollbars' );
+		window.open( xUrl, 'x', 'width=550,height=420,resizeable,scrollbars' );
```

Comment:
Currently we are opening new window on top left corner of the browser, can we open the new window on center of the browser? It will look better 🤔

Reply:
Done!
I also applied the fix to the Facebook popup.